src/zgamma_producer.cpp

```diff
@@ -49,21 +54,31 @@ void ZGammaVarProducer::WriteZGammaVars(nano_tree &nano, pico_tree &pico, vector
       pico.out_llphoton_deta().push_back(fabs(dilep.Eta() - photon.Eta()));
       pico.out_llphoton_iph().push_back(igamma);
       pico.out_llphoton_ill().push_back(ill);
+
+
       if(sig_jet_nano_idx.size() > 1) {
         TLorentzVector j1, j2, dijet;
         j1.SetPtEtaPhiM(nano.Jet_pt()[sig_jet_nano_idx[0]], nano.Jet_eta()[sig_jet_nano_idx[0]], nano.Jet_phi()[sig_jet_nano_idx[0]], nano.Jet_mass()[sig_jet_nano_idx[0]]);
         j2.SetPtEtaPhiM(nano.Jet_pt()[sig_jet_nano_idx[1]], nano.Jet_eta()[sig_jet_nano_idx[1]], nano.Jet_phi()[sig_jet_nano_idx[1]], nano.Jet_mass()[sig_jet_nano_idx[1]]);
         dijet = j1 + j2;
         pico.out_llphoton_dijet_dphi().push_back(llg.DeltaPhi(dijet));
         pico.out_llphoton_dijet_balance().push_back((dilep+photon+j1+j2).Pt()/(dilep.Pt()+photon.Pt()+j1.Pt()+j2.Pt()));
+        pico.out_llphoton_dijet_dr().push_back(llg.DeltaR(dijet));
         pico.out_photon_jet_mindr().push_back(min(photon.DeltaR(j1), photon.DeltaR(j2)));
+        pico.out_photon_jet_maxdr().push_back(max(photon.DeltaR(j1), photon.DeltaR(j2)));
+        pico.out_photon_jet1_dr().push_back(photon.DeltaR(j1));
+        pico.out_photon_jet2_dr().push_back(photon.DeltaR(j2));
         pico.out_photon_zeppenfeld().push_back(abs(photon.Eta() - (j1.Eta() + j2.Eta())/2));
-        TVector3 g = photon.Vect();
-        TVector3 h = llg.Vect();
-        TVector3 z = dilep.Vect();
-        g.SetZ(0); h.SetZ(0); z.SetZ(0);
-        pico.out_llphoton_pTt2().push_back(h.Cross(z-g).Mag()/h.Mag());
+
       }
+
+      TVector3 g_pT = photon.Vect();
+      TVector3 h_pT = llg.Vect();
+      TVector3 z_pT = dilep.Vect();
+      g_pT.SetZ(0); h_pT.SetZ(0); z_pT.SetZ(0);
+      pico.out_llphoton_pTt().push_back( h_pT.Cross((z_pT-g_pT).Unit()).Mag() );
+      pico.out_llphoton_pTt_an_hig019014().push_back( h_pT.Unit().Cross(z_pT-g_pT).Mag() );
```

Comment:
Why is "h_pT.Unit().Cross(z_pT-g_pT).Mag()" different with "h.Cross(z-g).Mag()/h.Mag()" that was there before?

Reply:
This is the same variable with a slightly different way of writing it, no? This way seems like a bit more clear in the difference between pTt and pTt_an_hig019014. I can move both to be written either way, but being consistent may be nice.